app-typescript/components/Avatar.tsx

```diff
@@ -12,7 +13,15 @@ interface IImageAvatar extends IBase {
 interface ITextAvatar extends IBase {
     text: string; // limited to 3 characters
 }
-
+interface INumberAvatar extends IBase {
+    number: string;
+}
+interface IIconAvatar extends IBase {
+    status?: 'default' | 'in-progress' | 'done';  // defaults to default
+    type?: 'text' | 'photo' | 'video' | 'audio' | 'file' | 'post' | 'post' | 'list-alt' | 'graphic';
```

Comment:
What is the benefit here of restricting what icons can be used with an Avatar?
I know we don't have other use cases right now, but would it be beneficial to allow any icon here (and remove the `canceled` prop, and the Planning module will choose which icon based on desired coverage assignment state)?